mkosi

```diff
@@ -4443,9 +4453,9 @@ def load_args(args: CommandLineArguments) -> CommandLineArguments:
         if args.output_format.is_disk():
             args.output = ('image' +
                            ('.qcow2' if args.qcow2 else '.raw') +
-                           ('.xz' if args.xz else ''))
+                           ('.xz' if args.xz else '.zst' if args.zstd else ''))
         elif args.output_format == OutputFormat.tar:
-            args.output = "image.tar.xz"
+            args.output = ('image.tar' + ('.xz' if args.xz else '') + ('.zst' if args.zstd else ''))
```

Comment:
How about this since they are mutually exclusive? 
```suggestion
            suffix = "zst" if args.zstd else "xz"
            args.output = f"image.tar.{suffix}"
```
I think it's easier to see what's happening. Alternatively, 
```suggestion
            args.output = "image.tar" + ".xz" if args.xz else ".zst" if args.zstd else ""
```
to mirrow the construction right above it.

Reply:
fixed